package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Command line interface for JavaAutolab",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha test"
+    "lint": "jshint **.js",
```

Comment:
@prasadtalasila Are both eslint and jshint needed? As far as I know, eslint would suffice as it has all the features of jshint, if not more. **.js was expected to be a wildcard for all js files in chid directories, but is not working.
@utkarshrai Instead of putting `jshint **.js`, put this `eslint test/*.js lib/*.js *.js`. Also the problem with travis build can be solved by putting `"test": "node index.js && ./node_modules/mocha/bin/mocha test"` instead of the one present.